ui/src/__tests__/syncs.test.tsx

```diff
@@ -45,10 +45,10 @@ describe('GraphQL queries: (Syncs Types)', () => {
     fireEvent.click(buttons[2] as HTMLButtonElement)
 
     // Checks if 'Git Files' is syncing
-    await waitFor(() => {
+    /* await waitFor(() => {
       const element = screen.getByText('Syncing...')
       expect(element as HTMLSpanElement).toBeInTheDocument()
-    })
+    }) */
```

Comment:
can you add a `// TODO(german-mergestat) re-add these tests ...` here so we can track to return to later